modules/sdk-coin-eth/src/lib/utils.ts

```diff
@@ -552,15 +562,29 @@ export function hasSignature(txData: TxData): boolean {
   );
 }
 
+type RecursiveBufferOrString = string | Buffer | RecursiveBufferOrString[];
+
 /**
  * Get the raw data decoded for some types
  *
  * @param {string[]} types ABI types definition
  * @param {Buffer} serializedArgs encoded args
  * @returns {Buffer[]} the decoded raw
  */
-export function getRawDecoded(types: string[], serializedArgs: Buffer): Buffer[] {
-  return EthereumAbi.rawDecode(types, serializedArgs);
+export function getRawDecoded(types: string[], serializedArgs: Buffer): RecursiveBufferOrString[] {
+  function normalize(v: any, i: number): any {
```

Comment:
```suggestion
  function normalize(v: unknown, i: number): unknown {
```
any turns off the type checking for that variable, more correct to use unknown as it  forces you to typecheck before use (Which you are doing btw)